### DISCUSSION TITLE
UGS not opening - possible JAVA issue?

### DISCUSSION THREAD

**Discussion by RostrevorMensShed** [winder/Universal-G-Code-Sender#2641] [Help]
We had been using UGS for around a year. Then last week, it would not connect to the CNC. Had a few issues previously whereby a reinstall solved issues, so we tried that. This time a reinstall won't work. We have deleted the .ugs file from C/Users/User, deleted the file, downloaded again a few times, but it still won't open. We have tried downloading the files with and without Java, and installing Java seperately. Still didn't work. Uninstalled Java and tried the combined download again but nothing works. The file looks like it is opening as normal, but getw about 2/3 the way across and then we just get a white screen. Each time we remove, the files, we reboot the computer but nothing is working. On Windows 10.

**Comment by breiler** [comment]
Without the logs it is hard to know what is going on. You can locate them under:
`C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log`

A common problem is that the settings are corrupt and can be found under this folder. Remove it and restart UGS to see if that helps:
`C:/Users/[your username>]/.ugsplatform/`

**Comment by RostrevorMensShed** [comment]
Thanks
There are three logs but only one contains text, which is as follows.

>Log Session: Tuesday, November 19, 2024 at 7:13:10 PM Greenwich Mean Time
>System Info: 
  Product Version         = Universal Gcode Sender 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44
  Operating System        = Windows 10 version 10.0 running on amd64
  Java; VM; Vendor        = 17.0.8.1; OpenJDK 64-Bit Server VM 17.0.8.1+1; Eclipse Adoptium
  Runtime                 = OpenJDK Runtime Environment 17.0.8.1+1
  Java Home               = C:\Users\User\Desktop\ugsplatform-win\jdk\jdk-17.0.8.1+1-jre
  System Locale; Encoding = en_GB (ugsplatform); Cp1252
  Home Directory          = C:\Users\User
  Current Directory       = C:\Users\User\Desktop\ugsplatform-win
  User Directory          = C:\Users\User\AppData\Roaming\ugsplatform
  Cache Directory         = C:\Users\User\AppData\Local\ugsplatform\Cache\var\cache
  Installation            = C:\Users\User\Desktop\ugsplatform-win\extra
                            C:\Users\User\Desktop\ugsplatform-win\etc
                            C:\Users\User\Desktop\ugsplatform-win\ide
                            C:\Users\User\Desktop\ugsplatform-win\ugsplatform
                            C:\Users\User\Desktop\ugsplatform-win\platform
                            C:\Users\User\Desktop\ugsplatform-win\java
                            C:\Users\User\Desktop\ugsplatform-win\bin
                            C:\Users\User\Desktop\ugsplatform-win\platform
  Boot & Ext. Classpath   = 
  Application Classpath   = C:\Users\User\Desktop\ugsplatform-win\platform\lib\boot.jar;C:\Users\User\Desktop\ugsplatform-win\platform\lib\org-openide-modules.jar;C:\Users\User\Desktop\ugsplatform-win\platform\lib\org-openide-util-lookup.jar;C:\Users\User\Desktop\ugsplatform-win\platform\lib\org-openide-util-ui.jar;C:\Users\User\Desktop\ugsplatform-win\platform\lib\org-openide-util.jar
  Startup Classpath       = C:\Users\User\Desktop\ugsplatform-win\platform\core\asm-9.5.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\asm-commons-9.5.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\asm-tree-9.5.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\core-base.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\core.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\org-netbeans-libs-asm.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\org-openide-filesystems-compat8.jar;C:\Users\User\Desktop\ugsplatform-win\platform\core\org-openide-filesystems.jar;C:\Users\User\Desktop\ugsplatform-win\ugsplatform\core\locale\core_ugsplatform.jar
-------------------------------------------------------------------------------
INFO [com.willwinder.universalgcodesender.model.GUIBackend]: Applying settings.
INFO [com.willwinder.universalgcodesender.model.GUIBackend]: Applying new command processor RunFromProcessor
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Starting UGS 2.1.9
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading LocalizingService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading JogService...
INFO [com.willwinder.universalgcodesender.utils.KeepAwakeUtils]: Moving the mouse location slightly to keep the computer awake.
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading OverrideActionService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading MacroService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading SendStatusLineService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading SettingsChangedNotificationService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading WindowTitleUpdaterService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading PendantService...
INFO [com.willwinder.ugs.nbp.core.services.PendantService]: Starting pendant service
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading ConsoleNotificationService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Loading FileFilterService...
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Services loaded!
INFO [com.willwinder.ugs.nbp.core.lifecycle.ServiceStarter]: Setting UGP version title.
INFO [org.netbeans.core.startup.NbEvents]: Turning on modules:
	org.openide.util.lookup [8.54 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.util [9.28 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.util.ui [9.29 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.modules [7.68 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.annotations.common/1 [1.48 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.filesystems [9.33 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.awt [7.88 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.progress/1 [1.68 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.progress.nb [1.68 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.dialogs [7.66 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.nodes [7.65 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.windows [6.97 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.batik.read [1.13.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.util.ui.svg [1.14 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.mimelookup/1 [1.60 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.text [6.88 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.scripting [1.17 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.swing.tabcontrol [1.77 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.swing.outline [1.54 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.explorer [6.82 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.actions [6.59 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.queries/1 [1.63 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.templates [1.28 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.filesystems.nb [9.30 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.loaders [7.90 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.intent [1.22 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.io [1.23 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.io [1.69 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.openide.filesystems.compat8 [9.29 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.swing.plaf [1.62 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.keyring [1.44 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.asm [5.23 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.bootstrap/1 [2.99 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.startup.base [1.82.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.startup/1 [1.82.0.1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.sampler [1.34 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core/2 [3.71 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.spi.quicksearch [1.47 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.options.api/1 [1.65 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.flatlaf/1 [1.14 3.1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.swing.laf.flatlaf [1.13 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.swing.laf.dark/1 [2.15.0.2 2 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.spi.tasklist/1 [1.55.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.spi.navigator/1 [1.58 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.settings/1 [1.77 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.util/1 [1.85 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.document [1.29.0.3 3 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.lexer/2 [1.83.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.lib2/1 [2.40.0.55.3 55 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.lsp/1 [1.15 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.settings.lib [1.71.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.fold/1 [1.65 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.indent/2 [1.63 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.guards/1 [1.53 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.lib/3 [4.27.0.23.3.55 23 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.spi.editor.hints/0 [1.62.0.8.55 8 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.templates/1 [1.27 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.settings/1 [1.69 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.sendopts/2 [2.56 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.options.keymap [1.57 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.settings.storage/1 [1.72.0.1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.fold.nbui [1.32.0.55 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor/3 [1.107.0.6.3.23.55 6 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.options.editor/1 [1.80 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.errorstripe.api/1 [2.52.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.errorstripe/2 [2.54.0.1.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.projectapi/1 [1.91 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.diff/1 [1.71.0.42.1 42 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.java.classpath/1 [1.74 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.lucene/3 [3.39 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.masterfs/2 [2.75.0.2 2 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.parsing.lucene/2 [2.56.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.parsing.api/1 [9.27.0.8 8 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.projectuiapi.base/1 [1.106.0.9 9 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.project.indexingbridge [1.37 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.parsing.indexing [9.29.0.1.3.8 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.refactoring.api [1.67.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.projectapi.nb [1.24 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.progress.ui [1.53 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.print [7.46 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.parsing.nb [1.24.0.8.1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.jna/2 [2.15 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.masterfs.windows [1.36 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.masterfs.ui [2.23.0.2 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.masterfs.nio2 [1.35 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.keyring.fallback [1.28 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.keyring.impl [1.44 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.ui/1 [1.65 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.api.search [1.41 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.jumpto/1 [1.75.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.janitor [1.13 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.favorites/1 [1.66 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.completion/1 [1.65.0.2 2 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.search [1.45.0.0.55 0 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.plain.lib/1 [1.53 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.plain/2 [1.55 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.mimelookup.impl/1 [1.52 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.codetemplates/1 [1.64.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.breadcrumbs/0 [1.37 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.editor.actions/1 [1.51.0.55 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.csl.types/1 [1.20 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.multiview/1 [1.64 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.csl.api/2 [2.78.0.2.1.1.8.1 2 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.autoupdate.services [1.76 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.autoupdate.cli [1.34 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.windows/2 [2.105 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.osgi [1.42 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.felix [2.35 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.autoupdate.ui [1.66 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.modules.core.kit [1.51 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.libs.jna.platform/2 [2.15 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.output2/1 [1.62 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.network [1.32 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.netigso [1.51 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.nativeaccess/1 [1.51 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.multitabs/1 [1.32.0.1 1 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	org.netbeans.core.io.ui/1 [1.50 18-7d33005615b3c744564f3d38ab9ef14bf8c3ec44]
	com.willwinder.ugs.platform.ugslib [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.ugscore [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.visualizer [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.welcome.page [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.surfacescanner [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.plugin.workflow [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.plugin.toolbox [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.setup.wizard [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.plugin.joystick [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.jog [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.plugin.filebrowser [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.dro [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.designer [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.console [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.plugin.cloud.storage [2.0.0.2024.9.3.18.52 2.1.9 202409031856]
	com.willwinder.ugs.platform.gcode.editor [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.ugs.platform.branding [2.0.0.2024.9.3.18.52 2.1.9 202409031854]
	com.willwinder.ProbeModule [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
	com.willwinder.GcodeTools [2.0.0.2024.9.3.18.52 2.1.9 202409031855]
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy resolver: Windows
INFO [org.netbeans.core.network.proxy.windows.WindowsNetworkProxy]: Windows system proxy resolver: auto detect
INFO [org.netbeans.core.network.proxy.windows.WindowsNetworkProxy]: Windows system proxy resolver: auto - PAC (http://wpad/wpad.dat)
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy reloading succeeded.
WARNING [org.openide.filesystems.Ordering]: Not all children in Menu/ marked with the position attribute: [Machine, Program, Visualizer], but some are: [File, Edit, Tools, Window, Help]
WARNING [org.openide.filesystems.Ordering]: Found same position 100 for both Toolbars/File and Toolbars/StartPauseStop
WARNING [org.openide.filesystems.Ordering]: Found same position 10 for both Toolbars/File/com-willwinder-ugs-nbp-core-actions-OpenAction.shadow and Toolbars/File/com-willwinder-ugs-nbp-core-actions-SaveAction.shadow
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: Initializing firmware... ...
INFO [org.openide.awt.Toolbar]: Too long AWTTask: 729 ms for org.openide.awt.Toolbar$Folder@1c82782(FolderList{MultiFileObject@11658a7[Toolbars/Connection]})
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/smoothieboard.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/grbl_esp32.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/g2core.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/tinyg.json
INFO [org.netbeans.core.network.proxy.ProxyAutoConfig]: InputStream for http://wpad/wpad.dat throws 
java.net.UnknownHostException: wpad
	at java.base/sun.nio.ch.NioSocketImpl.connect(Unknown Source)
	at java.base/java.net.Socket.connect(Unknown Source)
	at java.base/java.net.Socket.connect(Unknown Source)
	at java.base/sun.net.NetworkClient.doConnect(Unknown Source)
	at java.base/sun.net.www.http.HttpClient.openServer(Unknown Source)
	at java.base/sun.net.www.http.HttpClient.openServer(Unknown Source)
	at java.base/sun.net.www.http.HttpClient.<init>(Unknown Source)
	at java.base/sun.net.www.http.HttpClient.New(Unknown Source)
	at java.base/sun.net.www.http.HttpClient.New(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.AccessController.doPrivilegedWithCombiner(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.AccessController.doPrivilegedWithCombiner(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at org.netbeans.core.network.proxy.ProxyAutoConfig.downloadPAC(ProxyAutoConfig.java:155)
[catch] at org.netbeans.core.network.proxy.ProxyAutoConfig.initEngine(ProxyAutoConfig.java:94)
	at org.netbeans.core.network.proxy.ProxyAutoConfig$1.run(ProxyAutoConfig.java:82)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1419)
	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:45)
	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:287)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2034)
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/fluidnc.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/grbl.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/test_2.json
INFO [com.willwinder.universalgcodesender.utils.FirmwareUtils]: /resources/firmware_config/test_1.json
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy - mode: auto
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy - pac url: http://wpad/wpad.dat
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy TEST - http host: 
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy TEST - http port: 0
INFO [org.openide.awt.Toolbar]: Too long AWTTask: 1,242 ms for org.openide.awt.Toolbar$Folder@16634530(FolderList{MultiFileObject@79a8049e[Toolbars/File]})
INFO [FirmwareAction]: Changed to firmware GRBL
INFO [org.openide.awt.Toolbar]: Too long AWTTask: 955 ms for org.openide.awt.Toolbar$Folder@6e718076(FolderList{MultiFileObject@4be058c9[Toolbars/StartPauseStop]})
INFO [org.openide.awt.Toolbar]: Too long AWTTask: 1,033 ms for org.openide.awt.Toolbar$Folder@77193a35(FolderList{MultiFileObject@7f13ef48[Toolbars/Machine Actions]})
INFO [org.netbeans.ui.metrics.laf]: USG_LOOK_AND_FEEL
SEVERE [global]
java.lang.IllegalArgumentException: Key contains code point U+0000
	at java.prefs/java.util.prefs.AbstractPreferences.get(Unknown Source)
	at java.prefs/java.util.prefs.AbstractPreferences.getBoolean(Unknown Source)
	at org.netbeans.core.windows.TopComponentTracker.load(TopComponentTracker.java:76)
	at org.netbeans.core.windows.PersistenceHandler.load(PersistenceHandler.java:101)
	at org.netbeans.core.windows.WindowSystemImpl.load(WindowSystemImpl.java:56)
	at org.netbeans.core.GuiRunLevel$InitWinSys.run(GuiRunLevel.java:200)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
[catch] at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:136)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
INFO [com.willwinder.universalgcodesender.utils.KeepAwakeUtils]: Moving the mouse location slightly to keep the computer awake.
INFO [null]: Last record repeated again.
INFO [null]: Last record repeated more than 10 times, further logs of this record are ignored until the log record changes.

**Reply by breiler** [reply]
Your configuration seems to be corrupt, delete this folder: `C:\Users\User\AppData\Roaming\ugsplatform`
